package.json

```diff
@@ -16,7 +16,11 @@
     "test:browser": "plugin-helpers test:browser",
     "test:jest": "../../node_modules/.bin/jest --config ./test/jest.config.js",
     "build": "plugin-helpers build",
-    "release": "./tasks/release.sh"
+    "release": "./tasks/release.sh",
+    "start-cy-server": "yarn start --no-base-path --oss",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "e2e:test": "WAIT_ON_TIMEOUT=900000 start-server-and-test 'yarn start-cy-server' http-get://localhost:5601/app/opendistro-anomaly-detection-kibana 'yarn cy:run'"
```

Comment:
do we test by running `yarn e2e:test`?

Reply:
In the upcoming CI change, it will be used.

Reply:
Got it, makes sense